.travis.yml

```diff
@@ -11,15 +11,9 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-before_script:
-    - if [[ $deps = low ]]; then composer update --prefer-lowest --prefer-stable; fi
+install:
```

Comment:
Could we add `composer validate` as the first command as it validates that `composer.json` is valid.